### DISCUSSION TITLE
Function documentation

### DISCUSSION THREAD

**Discussion by cwolock** [statdivlab/rigr#23] [General]
I thought it would be useful to get people's thoughts on function documentation in rigr. 

Disclaimer: The code is still relatively new to me, so it's entirely possible I'm missing something!

I'm seeing somewhat inconsistent function documentation styles. For example, `wilcoxon.R` has something like the following format for each function argument: 

`@param my_parameter \code{dots} Here's a description of the parameter.`

Other functions, e.g. `scatter.R`, repeat the argument name: 

`@param my_parameter \code{my_parameter} Here's a description of the parameter.` 

Finally, other functions use what I consider to be fairly standard, which is just

`@param my_parameter Here's a description of the parameter.` 

This last style agrees with Hadley Wickham's R packages book, for whatever that's worth. 

Do people have thoughts on this? Again, there could be good reasons for the differing styles that I'm not aware of. But if not, I'd be in favor of using the final option above.

**Comment by bdwilliamson** [comment]
Thanks for compiling this. I'm in favor of using the final option!

**Comment by taylorokonek** [comment]
Also in favor of the final option
